apps/server/server.cpp

```diff
@@ -492,7 +500,15 @@ void invoke_sdk_api(payload::ClientRequest buff_recv) {
 
     case GET_FRAME: {
         aditof::Status status = aditof::Status::OK;
+        if (sameFrameEndlessRepeat) {
```

Comment:
Where is send the frame data?

Reply:
There's a global variable holding the frame to be sent (via network) to the host PC. It's called: 'buff_frame_to_send'. The content from it will be sent over and over again.